l10n_ch_import_cresus/README.rst

```diff
@@ -1,12 +1,12 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :target: http://www.gnu.org/licenses/agpl
    :alt: License: AGPL-3
 
 =============
-Import Crésus
```

Comment:
As all the text is in english, and accents don't exist in english, it did feel weird.